Lib/test/test_optimizer.py

```diff
@@ -43,6 +43,7 @@ def test_set_eval_frame_func(self):
         _testinternalcapi.set_eval_frame_default()
 
     def test_builtin_dict(self):
+        _testinternalcapi.reset_rare_event_counters()
```

Comment:
There are multiple other places where `get_rare_event_counters` is used. Why only a sinlge place handles reset?

Reply:
Yep, I'm talking about the same in https://github.com/python/cpython/pull/115128#issuecomment-1936252960.
I'm going to add reset in other places.